Sources/MessagingPush/MessagingPush.swift

```diff
@@ -7,10 +7,10 @@ import Foundation
   */
 public class MessagingPush: ModuleTopLevelObject<MessagingPushInstance>, MessagingPushInstance {
     @Atomic public private(set) static var shared = MessagingPush()
+    @Atomic public private(set) static var moduleConfig: MessagingPushConfigOptions = MessagingPushConfigOptions.Factory.create()
```

Comment:
Since we've already implemented this in the upcoming changes for improving push metrics, I've reused the same to avoid introducing another option for the same functionality.